zap-hooks.py

```diff
@@ -0,0 +1,5 @@
+def zap_started(zap, target):
+  zap.pscan.set_scanner_alert_threshold(id='90033', alertthreshold='OFF')
+  zap.pscan.set_scanner_alert_threshold(id='10023', alertthreshold='OFF')
+  zap.pscan.set_scanner_alert_threshold(id='10010', alertthreshold='OFF')
+  zap.pscan.set_scanner_alert_threshold(id='10054', alertthreshold='OFF')
```

Comment:
Do you think something like this is easier to extend or do you think that the additional clarity provided by listing each out is better?
```suggestion
def zap_started(zap, target):
  rules_to_ignore = ('90033', '10023', '10010', '10054')
  for rule in rules_to_ignore:
    zap.pscan.set_scanner_alert_threshold(id=rule, alertthreshold='OFF')
```

Reply:
yeah, why not, done